spaces/S000152/README.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: S000152
+name: Poset $\{-1,0_a,0_b\}\cup\{1/n\}_{n=1}^\infty$ with Alexandrov topology
+refs:
+  - wikipedia: Alexandrov_topology
+    name: Alexandrov topology on Wikipedia
+  - mathse: 5007860
+    name: Answer to "For a compact sober "highly non-$T_1$" space, how much "highly connectedness" is needed to imply it's a spectral space?"
+---
+
+Consider the set $\{1/n\}_{n=1}^\infty$ with its usual order. Let,
+
+$\quad X=\{-1,0_a,0_b\}\cup\{1/n\}_{n=1}^\infty$
+
+where either of the zeros is smaller than all $1/n$, the two zeros are incomparable, and $-1$ is smaller than everything else. Then equip $X$ with the Alexandrov topology associated to the partial order defined above.
```

Comment:
```suggestion
where either of the zeros is smaller than all $1/n$, the two zeros are incomparable, and $-1$ is smaller than everything else. Then equip $X$ with the Alexandrov topology associated to the partial order defined above; that is, the open sets are the upper sets for $\le$.
```
adding this clarification, because some books define the open sets to be the lower sets.